### DISCUSSION TITLE
Is it possible to see the progress of docker image pooling?

### DISCUSSION THREAD

**Discussion by obalunenko** [ory/.github#35] [Q&A]
Is it possible to see the progress of docker image pooling during [Run](https://pkg.go.dev/github.com/ory/dockertest/v3#Pool.Run) and [RunWithOptions](https://pkg.go.dev/github.com/ory/dockertest/v3#Pool.RunWithOptions)?

**Comment by vinckr** [comment]
Hello @obalunenko 
Sorry must have missed this thread. 

Did you find a solution since?
